packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx

```diff
@@ -145,7 +173,7 @@ const TopBarSearchModal = () => {
               <input
                 type="text"
                 className={styles._MainInput}
-                placeholder={t("search_placeholder")}
+                placeholder={t("Search")}
```

Comment:
same to this. Don't edit it out. It was added there for a reason

Reply:
Noted